packages/discovery-react-components/src/components/DocumentPreview/utils/documentData.ts

```diff
@@ -42,16 +43,21 @@ export function isJsonFile(doc: QueryResult | null | undefined): boolean {
 /**
  * Returns the preview type for document
  */
-export function detectPreviewType(document: DiscoveryDocument, file?: DocumentFile): PreviewType {
+export function detectPreviewType(
+  document: DiscoveryDocument,
+  highlight: QueryResultPassage | QueryTableResult | undefined,
```

Comment:
Instead of giving this a possible type of `undefined`, just make it an optional type:
```suggestion
  highlight?: QueryResultPassage | QueryTableResult,
```

That way, you don't need to add a 2nd `undefined` param to all of the tests, etc.